BooruSharp/Booru/ABooru.cs

```diff
@@ -300,6 +324,8 @@ protected get
                 // Add our User-Agent if client's User-Agent header is empty.
                 if (_client != null && !_client.DefaultRequestHeaders.Contains("User-Agent"))
                     _client.DefaultRequestHeaders.Add("User-Agent", _userAgentHeaderValue);
+
+                // TODO: Need to setup HttpClientHandler but I don't know how
```

Comment:
We don't need to since it's on a developer who sets their own `HttpClient` to also properly initialize their own handler.

Reply:
I added a comment with 1e450f639534699d370924cca81db13c66ce5294